opencensus/trace/propagation/google_cloud_format.py

```diff
@@ -107,7 +110,7 @@ def to_header(self, span_context):
 
         header = '{}/{};o={}'.format(
             trace_id,
-            span_id,
+            int(span_id, 16),
```

Comment:
We probably need to update `from_header` as well.

@c24t FYI - this is a potential breaking change (e.g. if customers mix use old version and the latest version of OpenCensus Python SDK).

Reply:
the from_header has been updated, it contains

```python
            if span_id:
                span_id = '{:016x}'.format(int(span_id))
```

to do the conversion, or do you mean something additional?

Reply:
@mhindery please update `CHANGELOG.md` if this is a breaking change. Thanks.

Reply:
Adding @c24t to comment from Google side.

I cannot find the official spec for `X-Cloud-Trace-Context`, and I'm a bit worried that this change might break existing stuff for Google (considering there are system tests running as part of the CI, it is a bit surprising to me that we've been sending the wrong format to Stackdriver).